tests/acceptance/features/apiSpaces/notification.feature

```diff
@@ -0,0 +1,126 @@
+@api @skipOnOcV10
+Feature: Notification
+  As a user
+  I want to be notified of actions related to me
+
+  Background:
+    Given these users have been created with default attributes and without skeleton files:
+      | username |
+      | Alice    |
+      | Brian    |
+      | Carol    |
+    And the administrator has given "Alice" the role "Space Admin" using the settings api
+
+
+  Scenario: user gets a notification of space sharing
+    Given user "Alice" has created a space "notificaton checking" with the default quota using the GraphApi
+    And user "Alice" has shared a space "notificaton checking" with settings:
+      | shareWith | Brian  |
+      | role      | editor |
+    When user "Brian" list all notifications
+    Then the HTTP status code should be "200"
+    And the JSON response should contain message with type "Space shared" and match
+    """
+    {
+    "type": "object",
+      "required": [
+        "app",
+        "datetime",
+        "message",
+        "messageRich",
+        "messageRichParameters",
+        "notification_id",
+        "object_id",
+        "object_type",
+        "subject",
+        "subjectRich",
+        "user"
+      ],
+      "properties": {
+        "app": {
+          "type": "string",
+          "enum": ["userlog"]
+        },
+        "message": {
+          "type": "string",
+          "enum": ["Alice Hansen added you to Space notificaton checking"]
+        },
+        "messageRich": {
+          "type": "string",
+          "enum": ["{user} added you to Space {space}"]
+        },
+        "messageRichParameters": {
+          "type": "object",
+          "required": [
+            "space",
+            "user"
+          ],
+          "properties": {
+            "space": {
+              "type": "object",
+              "required": [
+                "id",
+                "name"
+              ],
+              "properties": {
+                "id": {
+                  "type": "string"
```

Comment:
I don't check spaceUuid here. maybe this is unnecessary since there is also a name check

Reply:
I check regex here in the feature file. I took an obscene amount of time to check this in featureContext but couldn't